theme/material/multiselect-combo-box-styles.js

```diff
@@ -37,5 +37,5 @@ const multiSelectComboBox = css`
 `;
 
 registerStyles('multiselect-combo-box', [inputFieldShared, multiSelectComboBox], {
-  moduleId: 'multiselect-combo-box'
+  moduleId: 'material-multiselect-combo-box'
```

Comment:
A material- prefix was also missing here, so I added it😉

Reply:
Awesome, thank you!